Clean up edits (#1312)
mulesoft/docs-connectors: hl7-mllp/2.1/modules/ROOT/pages/hl7-mllp-connector-examples.adoc

= HL7 EDI - Examples - Mule 4

> Use these examples to

Reviewer: Need an intro sentence. Use these examples to... ?

Author: Opps, I made the change but forgot to push it before handing off the PR. Here it is:
Use these examples to read and write HL7 messages over the MLLP protocol.

== Example: HL7 Studio

Set these Global properties before using the examples:

* Delimiter: The standard HL7 delimiter value that separates

> MLLP messages. It is a hexadecimal 0x1C character also known as FS. Applications in some clinics use a different delimiter value, that can be set this field. The standard value in the

Reviewer: ```suggestion
MLLP messages. It is a hexadecimal 0x1C character also known as FS. Applications in some clinics use a different delimiter value, that can be set in this field. The standard value in the
```

Author: Fixed.

HL7 MLLP connector is shown as `001C`.
+
* Host: Sets the socket host address.
* Port: Sets the socket port.
+
For an advanced connection, complete these fields:
+
* Reuse address: Whether to reuse the configured socket or fail when trying to bind to the socket.
* Send TCP With No Delay: Whether or not to transmit data immediately, or collect data for greater efficiency before sending.
* Linger: How long in milliseconds that the socket takes to close so any remaining data transmits correctly.
* Keep alive: Whether to close open socket connections that are unused for a long period and with an unavailable connection.
* Fail on unresolved host: Whether the socket should fail during its creation if the host set on the endpoint cannot be resolved.

> * In the TLS tab you can configure a new TLS configuration, its presence implies the use of SSLServerSocket instead of plain TCP.

Reviewer: ```suggestion
* In the TLS tab you can configure a new TLS configuration. The presence of the TLS tab implies the use of SSL Server Socket instead of plain TCP.
```

Reviewer: It's usually SSL Socket Server, but it wouldn't surprise me if it's wrong in the UI. You should probably verify.

Author: It is correct.

Author: SSLServerSocket

> * You can also set the send or receive buffer size.

Reviewer: ```suggestion
* Set the send or receive buffer size.
```

Author: Fixed.
